week04/Design Linked List.cpp

```diff
@@ -0,0 +1,65 @@
+//TC=O(1)for each operation

```

Comment:
Really? If that's the case, what's the point of using `set` that has a logarithmic time complexity for each operation? :)

Reply:
@Sadman007 Sorry vaiya I didn't understand your point.